crates/aptos-protos/proto/aptos/transaction/testing1/v1/transaction.proto

```diff
@@ -326,20 +332,21 @@ message MoveFunctionGenericTypeParam {
 
 // Next id: 12
```

Comment:
is this value supposed to be bumped?

Reply:
removed. leave this to be handled by `buf lint`